tpch-scripts/TpchQueryExecute.sh

```diff
@@ -22,18 +21,25 @@ QUERY_DIR=$BENCH_HOME/$BENCHMARK/sample-queries-tpch
 
 RESULT_DIR=$BENCH_HOME/$BENCHMARK/results/
 
+PLAN_DIR=$BENCH_HOME/$BENCHMARK/plans/
+
 if [ ! -d "$RESULT_DIR" ]; then
 mkdir $RESULT_DIR
 chmod -R 777 $RESULT_DIR
 fi
 
+if [ ! -d "$PLAN_DIR" ]; then
+mkdir $PLAN_DIR
+chmod -R 777 $PLAN_DIR
+fi
+
 LOG_FILE_EXEC_TIMES="${BENCH_HOME}/${BENCHMARK}/logs/query_times.csv"
 
 if [ ! -e "$LOG_FILE_EXEC_TIMES" ]
   then
 	touch "$LOG_FILE_EXEC_TIMES"
 	chmod 777 $LOG_FILE_EXEC_TIMES
-	echo "QUERY,DURATION_IN_SECONDS,STARTTIME,STOPTIME,BENCHMARK,DATABASE,SCALE_FACTOR,FILE_FORMAT" >> "${LOG_FILE_EXEC_TIMES}"
+	echo "QUERY,DURATION_IN_SECONDS,STARTTIME,STOPTIME,BENCHMARK,DATABASE,SCALE_FACTOR,FILE_FORMAT,STATUS" >> "${LOG_FILE_EXEC_TIMES}"
```

Comment:
There was a mismatch between header and he actual data previously, right? Is that corrected here?